.eleventy.js

```diff
@@ -619,6 +619,19 @@ module.exports = function(eleventyConfig) {
         return nav;
     });
 
+    eleventyConfig.addCollection("ebooksAndWhitepapers", function(collectionApi) {
```

Comment:
Let's call it a library or something else, but this is not so nice

Reply:
I've changed it to `publications`